### PR TITLE
[#22] Packager section

### LINKED ISSUE #22: Packagers section

### PR DESCRIPTION
**Pull request by mcdoker18**
This closes #22

### CODE REVIEW

File: source/ref_packagers.rst [hunk 1]
```diff
@@ -0,0 +1,462 @@
...
+    # Every packager saves its results into status field.
+    # Example of fields: docker image or python packager name.
+    status:
+        results:
+            - name: some_param
+              value: some_value
+
+.. note::
+
+    You can find a artifactName in the `status.artifactName` field of a model training entity.
```
Comment (maxkochubey) [inline source/ref_packagers.rst]:
```suggestion
    You can find an artifactName in the `status.artifactName` field of a model training entity.
```

File: source/ref_packagers.rst [hunk 2]
```diff
@@ -0,0 +1,462 @@
...
+Swagger UI
+----------
+
+ModelPackaging and PackagingIntegration are available on the Swagger UI at http://api-service/swagger/index.html URL.
+
+Odahu-flow CLI
+--------------
+
+:ref:`ref_odahuflowctl:Odahuflowctl` supports the Packagers API.
+You must be :ref:`login <ref_odahuflowctl:Login>` if you want to get access to the API.
```
Comment (maxkochubey) [inline source/ref_packagers.rst]:
```suggestion
You must be :ref:`logged in <ref_odahuflowctl:Login>` if you want to get access to the API.
```

**Review by maxkochubey — APPROVED** [review]

File: source/ref_packagers.rst [hunk 3]
```diff
@@ -0,0 +1,462 @@
...
+   "docker-pull", ":ref:`docker<ref_connections:Docker>`, :ref:`ecr<ref_connections:Amazon Elastic Container Registry>`", "False", "The packager will use the connection for pulling a custom base Docker image"
+
+The packager provides the following list of arguments:
+
+.. csv-table::
+   :header: "Argument Name", "Type", "Default", "Required", "Description"
+   :widths: 20, 20, 20, 10, 100
+
+   "imageName", "string", "{{ Name }}-{{ Version }}:{{ RandomUUID }}", "False", "This option provides a way to specify the Docker image name. You can hardcode the full name or specify a template. Available template values: Name (Model Name), Version (Model Version), RandomUUID. Examples: myservice:123, {{ Name }}:{{ Version }}"
+   "port", "integer", "5000", "False", "Port to bind"
```
Comment (karbyshevds) [inline source/ref_packagers.rst:149]: params description says integer but you provide examples in quotes, is it correct ?

**Review by karbyshevds — CHANGES_REQUESTED** [review]

File: source/ref_packagers.rst [hunk 4]
```diff
@@ -0,0 +1,462 @@
...
+        "predictions"
+      ]
+    }
+
+
+********************************************
+Docker CLI
+********************************************
+
+The Docker REST packager wraps an ML model into the CLI inside a Docker image.
```
Comment (vlad-tokarev) [inline source/ref_packagers.rst]: not REST here

**Review by vlad-tokarev — APPROVED** [review]

File: source/ref_packagers.rst (hunk 3 above)
Comment (mcdoker18) [reply source/ref_packagers.rst:149]: Screenshot from 2019-12-13 13-44-59
Yeap, the result table does not contain any quotes.

**Pull request merged**